README.md

```diff
@@ -213,6 +213,35 @@ WMCO is not responsible for Windows operating system updates. The cluster admini
 creating the VMs and hence, the cluster administrator is responsible for providing an updated image. The cluster 
 administrator can provide an updated image by changing the image in the MachineSet spec.
 
+## Windows nodes Openshift EUS-to-EUS upgrade
+
+WMCO does support upgrading from one EUS version to another EUS version of OCP, the Windows nodes will be upgraded 
```

Comment:
It might be helpful to add a link for EUS, not required though.
 https://access.redhat.com/support/policy/updates/openshift-eus

Reply:
addressed https://github.com/openshift/windows-machine-config-operator/commit/a2f163ee7e0b14fe2ffa2bab605f3802782b0b98